tests/steamship_tests/client/operations/test_embed_file.py

```diff
@@ -29,36 +29,35 @@ def test_file_parse():
         steamship,
         content=content,
         mime_type=MimeTypes.MKD,
-    ).data
+    )
     assert file.id is not None
     assert file.mime_type == MimeTypes.MKD
 
     blockify_resp = file.blockify(plugin_instance="markdown-blockifier-default-1.0")
-    assert blockify_resp.error is None
     blockify_resp.wait()
 
     # Now we parse
-    parser = PluginInstance.create(steamship, plugin_handle="test-tagger").data
+    parser = PluginInstance.create(steamship, plugin_handle="test-tagger")
     parse_resp = file.tag(plugin_instance=parser.handle)
-    assert parse_resp.error is None
     parse_resp.wait()
 
     # Now the sentences should be parsed!
-    q2 = file.refresh().data
+    q2 = file.refresh()
     assert len(q2.blocks) == 6
 
     # Now we add the file to the index
-    plugin_instance = PluginInstance.create(steamship, plugin_handle=_TEST_EMBEDDER).data
+    plugin_instance = PluginInstance.create(steamship, plugin_handle=_TEST_EMBEDDER)
     with random_index(steamship, plugin_instance=plugin_instance.handle) as index:
         index.insert_file(file.id, reindex=False)
         embed_resp = index.embed()
-        assert embed_resp.error is None
         embed_resp.wait()
 
-        res = index.search("What color are roses?").data
-        assert len(res.items) == 1
+        res = index.search("What color are roses?")
+        res.wait()
+        items = res.output.items
```

Comment:
Going to go back and review the Task object, but just noting here aloud that async data now live at `task.output`. 👍